go/cmd/dolt/commands/tblcmds/import.go

```diff
@@ -520,7 +529,11 @@ func move(ctx context.Context, rd table.SqlRowReader, wr *mvdata.SqlEngineTableW
 	badRowCB := func(row sql.Row, err error) (quit bool) {
 		// record the first error encountered unless asked to ignore it
 		if row != nil && rowErr == nil && !options.contOnErr {
-			rowErr = fmt.Errorf("A bad row was encountered: %s: %w", sql.FormatRow(row), err)
+			if ue, ok := err.(sql.WrappedInsertError).Cause.(*errors.Error).Cause().(sql.UniqueKeyError); ok {
```

Comment:
I'm not thrilled about this line, but I think the improved error messaging is worth it, and this way doesn't touch GMS.

Reply:
I assumed that ok would be true if all the type conversions were successful, and ue would be nil if any of them failed. Turns out it doesn't work that way. Good to know.

Reply:
Fixed.